modules/ROOT/pages/ext-analytics-elk.adoc

```diff
@@ -0,0 +1,173 @@
+= Analyze Business and API Data using ELK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the Elastic Stack (ELK) to analyze the business data and API analytics generated by Anypoint Platform Private Cloud Edition (Anypoint Platform PCE). You can use Filebeat to process Anypoint Platform log files, insert them into an Elasticsearch database, and then analyze them with Kibana.
```

Comment:
I found this when I looked up ELK for our Term Presentation List:

The **ELK stack** is an acronym used to describe a stack that comprises of three popular open-source projects: Elasticsearch, Logstash, and Kibana.

Do you support my listing of this term as "ELK stack"?

Reply:
Is it okay that this narrative does not mention Logstash?

Reply:
Do we have anything to say about WHY your audience wants or needs these products and processes, or why they in particular are superior?